features/bevragen/persoon/adressering/adres-regels/onbekendwaardes.feature

```diff
@@ -0,0 +1,53 @@
+#language: nl
+
+Functionaliteit: samenstellen van adresregels wanneer één of meerdere verblijfplaats velden de standaardwaarde heeft
+
+  Als consumer van de BRP bevragen API
+  Wil ik dat de verblijfplaats gegevens van een persoon kan worden gevraagd conform NEN 5825:2002
+  Zodat ik de adresgegevens van de persoon op de correcte manier kan gebruiken 
+
+Rule: adresregels voor een adres worden niet geleverd als 'straatnaam (11.10)' of 'huisnummer (11.20)' de standaardwaarde als waarde hebben
+
+  Abstract Scenario: persoon heeft een adres met <omschrijving>
+    Gegeven de persoon met burgerservicenummer '000000188' heeft de volgende 'verblijfplaats' gegevens
+    | functie adres (10.10) |
+    | W                     |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | gemeentecode (92.10) | straatnaam (11.10) | huisnummer (11.20) |
+    | 0518                 | <straatnaam>       | <huisnummer>       |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                                                                                   |
+    | type                | RaadpleegMetBurgerservicenummer                                                          |
+    | burgerservicenummer | 000000188                                                                                |
+    | fields              | adressering.adresregel1,adressering.adresregel2,adressering.adresregel3,adressering.land |
+    Dan heeft de response een persoon zonder 'adressering' gegevens
+
+    Voorbeelden:
+    | straatnaam               | huisnummer | omschrijving                       |
+    |                          | 88         | een lege straatnaam                |
+    | .                        | 88         | een straatnaam met standaardwaarde |
+    | Jonkheer van Riemsdijkln | 0          | een huisnummer met standaardwaarde |
+
+Rule: adresregels voor een verblijfplaats buitenland worden niet geleverd als 'land (13.10)' de standwaarde heeft of 'regel 1 adres buitenland (13.30)', 'regel 2 adres buitenland (13.40)' en 'regel 3 adres buitenland (13.50)' hebben de standaardwaarde
```

Comment:
de regel velden (13.30-13.50) hebben geen standaardwaarde. Ze hebben een waarde of ze hebben geen waarde.

Reply:
aangepast